### PR TITLE
Personalised dashboard help center announcement banner

### PR DESCRIPTION
**Pull request by PippoRaimondiDIT**
## Description of change

Add a dismissible banner with announcement feed in the top of the new dashboard page.

## Test instructions

Functional tests have been added for the feature.

## Screenshots
Screenshot 2022-11-03 at 09 06 36
Screenshot 2022-11-03 at 09 04 53

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'

- [ ] Has the branch been rebased to main?
- [ ] Automated tests (Any of the following when applicable: Unit, Functional or End-to-End)
- [ ] Manual compatibility testing (Browsers: Chrome, Firefox, Edge, Safari)

### CODE REVIEW

File: test/functional/cypress/specs/dashboard-new/data-hub-feed-spec.js [hunk 1]
```diff
@@ -68,5 +69,48 @@ describe('Dashboard - Data Hub feed', () => {
...
+        .then(() => {
+          cy.wrap(localStorage.getItem('isBannerDismissed')).should(
+            'contain',
+            'https://test-url'
+          )
+        })
+    })
+  })
+
+  context('Has updates with banner cookie', () => {
```
Comment (paulgain) [inline test/functional/cypress/specs/dashboard-new/data-hub-feed-spec.js:91]: There is no cookie, we're simply writing to local storage.

File: test/functional/cypress/specs/dashboard-new/data-hub-feed-spec.js [hunk 2]
```diff
@@ -68,5 +69,48 @@ describe('Dashboard - Data Hub feed', () => {
...
+            link: 'https://test-url2',
+            date: '2 hours ago',
+          },
+        ],
+      }).as('apiRequest')
+      cy.visit('/')
+      cy.wait('@apiRequest')
+    })
+
+    it('should not display feed banner if cookie is set', () => {
```
Comment (paulgain) [inline test/functional/cypress/specs/dashboard-new/data-hub-feed-spec.js:112]: Same as above.

**Review by paulgain — COMMENTED** [review]
Looks good, just a few comments.

**Comment by PippoRaimondiDIT** [comment]
> Looks good, just a few comments.

Sorted everything, ty so much for the review!

**Review by paulgain — APPROVED** [review]

**Review by elcct — APPROVED** [review]

**Pull request merged**